### DISCUSSION TITLE
How to pass to data to GNN model?

### DISCUSSION THREAD

**Discussion by yancychy** [optuna/optuna#3897] [Q&A] [answered]

**Comment by nzw0301** [comment, marked as answer]
In fact, this bug is not related to Optuna at all. Anyway, as in the error message, the data don't have the attributes `x` and `edge_index`. Could you double check the input data sample has such attributes?

**Reply by yancychy** [reply]
Thanks. 
The data format is like this
```
Data(x=[2708, 1433], edge_index=[2, 10556], y=[2708], train_mask=[2708], val_mask=[2708], test_mask=[2708])
```
